app/code/community/EcomDev/PHPUnit/Model/Yaml/Loader.php

```diff
@@ -109,6 +109,6 @@ public function resolveFilePath($className, $type, $name)
      */
     public function load($filePath)
     {
-        return Spyc::YAMLLoad($filePath);
+        return \Symfony\Component\Yaml\Yaml::parse(file_get_contents($filePath));
```

Comment:
Bisognerebbe fare il require di symfony/yaml in composer.json. E poi se questo Spyc non è più usato da altre parti rimuoverei proprio il file e il require_once presente all'inizio di questo file.